plugin.video.francetv/addon.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.francetv" name="france•tv" version="1.0.0" provider-name="melmorabity">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="script.module.dateutil" version="2.5.3"/>
+    <import addon="script.module.requests" version="2.18.4"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">france•tv</summary>
```

Comment:
Not sure this will work accross all skins (I guess it depends on the fontset) and not sure it works in our addon website. Who should I ping? @MartijnKaijser

Reply:
@enen92 To avoid any potential problem, I will also update the add-on summary to avoid tofus with fonts without the "•" glyph.